### ISSUE TITLE
NextTask options for dropping off vehicles

### ISSUE THREAD

**Issue by joewhite94** [Global-Conflicts-ArmA/Olsen-Framework-Arma-3#300]
Currently vehicles have quite fixed behaviour following a dropoff task, with ground vehicles covering and air vehicles potentially RTBing.

We should refactor the Dropoff and/or Land tasks to allow MMs to configure a next task for vehicles after performing their dropoff, which replaces the default cover/rtb tasks.

**Comment by PiZZAD0X** [comment]
Yes I think assault/attack should have current mech vehicle behaviour and dropoff should be a transport task, drop off and fuck off
